Neos.Flow/Classes/Mvc/Exception/UnsupportedRequestTypeException.php

```diff
@@ -14,7 +14,7 @@
 /**
  * An "Unsupported Request Type" exception
  *
- * @api
+ * @deprecated since Flow 9.0
  */
 class UnsupportedRequestTypeException extends \Neos\Flow\Mvc\Exception
```

Comment:
i could not find any instantiation anymore, only imports and dead `@throws`. So it should be safe to remove this exception. (Certainly in another pr) What do you mean?

Reply:
But please in a separate PR 😅 otherwise this will become a slight big ball of diff stuff 😂